adaptors/library/jobs/commcare-create-user.js

```diff
@@ -0,0 +1,18 @@
+// Create a user and submit their case to commcare
+
+post('user', {
+  username: 'janedoe42',
+  password: 'qwer12345',
+  first_name: 'Jane',
+  last_name: 'Doe',
+  default_phone_number: '+50253311399',
+  email: 'jdoe@example.org',
+});
+
+submitXls([{ name: $.data.username, feeling_sick: 'No' }], {
```

Comment:
Just to check: presumably the `post` will write the newly created user to `state.data`?

Reply:
Yes. That is how it should work